util/json.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
I'm not sure about `util` as a package name, it seems very generic. Maybe `jsonutil` ? I know it only carries a single helper for now, but I am concerned that it will become a bit of a grab-all package if we scoope it so wide.